src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
+
+  const sexPerson = sex === 'f' ? 'husband' : 'wife';
```

Comment:
The condition for sexPerson should be reversed: 'f' should be 'wife' and 'm' should be 'husband'.